sorting/bubble_sort_recursion.c

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file
+ * @brief [Bubble sort](https://en.wikipedia.org/wiki/Bubble_sort) algorithm
+ * implementation
```

Comment:
to distinguish from the other implementation,
```suggestion
 * implementation using recursion.
```